### ISSUE TITLE
New Chat - There is NO focus when opening tab via FAB button

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#19290]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to URL https://staging.new.expensify.com/
2. Login with any account
3. Click on FAB button -> New Chat

## Expected Result:
Cursor presents in the search field

## Actual Result:
The cursor is missing in the search field.
New Group - the same behavior.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.16.3

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://platform.applause.com/services/links/v1/external/5bae2830f9fdeb089985848c3383dbcd2e3111818892065011d40b0c64502352

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by hungvu193** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
New Chat - There is NO focus when opening tab via FAB button
### What is the root cause of that problem?
We're using OptionSelector to show the TextInput and Options:
https://github.com/Expensify/App/blob/9e5b119f0005088a3870ab4eb53e6b4795d9b8d4/src/pages/NewChatPage.js#L240-L243
But since we implemented skeleton, the input will need a little bit times to focus.
### What changes do you think we should make in order to solve the problem?

We should add `shouldDelayFocus` to our OptionsSelector:
```javascript
  <OptionsSelector
                                // autoFocus
                                shouldDelayFocus={shouldDelayFocus}
```
### What alternative solutions did you explore? (Optional)
NA

**Comment by isabelastisser** [comment]
I will prioritize this today.

**Comment by isabelastisser** [comment]
I don't think this is a bug, this is expected behavior, and the cursor will appear when the user clicks on the `name, email, or phone number` field under `new chat` and `new group`.
